public-api/src/main/resources/public-api.yaml

```diff
@@ -339,6 +339,10 @@ definitions:
         type: "array"
         items:
           type: "string"
+      hasCounts:
```

Comment:
Do we need this in the API? seems like it should be easy for the client to check this for themselves using the current API model.

If not, I would suggest just keeping it an internal implementation detail

Reply:
Added it in there for tests to run, otherwise was not using it anywhere. Fixed it. Thanks.